generator/source/about.md

```diff
@@ -9,37 +9,85 @@ feat_img_size: small
 # https://creativecommons.org/licenses/by/4.0/
 ---
 
-# Hyperledger Grid
+Hyperledger Grid is a project to provide a set of shared, reusable tools for
+building cross-industry supply chain solutions based on distributed ledger
+platforms. Its goal is to simplify the development of distributed-ledger-based
+solutions for all types of supply chain scenarios.
 
-Hyperledger Grid is a project for building supply chain solutions. It includes
-a set of libraries, data models, and SDKs to accelerate development for supply
-chain smart contracts and client interfaces.
+This ecosystem of technologies, frameworks, and libraries is designed to
+work with existing distributed ledger platform software and business-specific
+applications, so that application developers can choose which components are
+the most appropriate for their industry or market model.
 
-# Documentation
+{: .center-img}
 
-Information on how to run and extend Hyperledger Grid is available at
-[https://grid.hyperledger.org/docs/](https://grid.hyperledger.org/docs/).
+![Hyperledger Grid in the supply change solution stack](/img/grid-diagram.png)
 
-# Project Status
+Hyperledger Grid components include:
 
-This Hyperledger project is in _incubation_. It was proposed to the community
-and documented in this [Hyperledger Grid
-Proposal](https://docs.google.com/document/d/1b6ES0bKUK30E2iZizy3vjVEhPn7IvsW5buDo7nFXBE0/edit).
-Information on what _incubation_ means can be found in the [Hyperledger Project
-Lifecycle document](https://wiki.hyperledger.org/community/project-lifecycle).
+* Domain-specific data models based on existing open standards, such as
+  GS1 product definitions that can be used in a product catalog
 
-# License
+* Reference implementations of supply-chain-centric data types
```

Comment:
This is redundant w/ prior bullet.  

I like:

Reference implementations of supply-chain-centric data types, including domain-specific models based on existing open standards, such as GS1.

Reply:
Rewrote